sdk/python/assets/data/data.ipynb

```diff
@@ -298,6 +258,275 @@
     "tbl.to_pandas_dataframe()"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import Data from external sources\n",
+    "\n",
+    "You can create a data asset in Azure Machine Learning by importing data from external sources like Snowflake or Amazon S3 and caching it, which has the following benefits:\n",
+    "\n",
+    "- All the benefits of caching including faster and more reliable access of data for the training jobs\n",
+    "- Less points of failure than connecting to data in external sources directly\n",
+    "- Easy to share with other members of the team (no need to remember file locations)\n",
+    "- Versioning of the metadata (location, description, etc) even for data from SQL/ relational database \n",
+    "- Lineage tracking\n",
+    "\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Firstly, you need to create a workspace connection with details on how to connect to the external source. \n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Creating a connection to Snowflake DB when you want to import data from Snowflake"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml import MLClient\n",
+    "from azure.ai.ml.entities import WorkspaceConnection\n",
+    "from azure.ai.ml.entities import UsernamePasswordConfiguration\n",
+    "\n",
+    "name = \"my_snowflakedb_connection\"\n",
+    "\n",
+    "target = \"jdbc:snowflake://<myaccount>.snowflakecomputing.com/?db=<mydb>&warehouse=<mywarehouse>&role=<myrole>\"\n",
```

Comment:
The target did not quite render correctly in the notebook. I think the `<>` placeholders cause some kind of issue.

Might consider using:

```python
snowflake_account_name="account_name"
db_name="db_name"
warehouse_name="warehouse_name"
role="my_role"

target = f'jdbc:snowflake://{account_name}.snowflakecomputing.com/?db={db_name}&warehouse={warehouse_name}&role={role}'
```